src/rules/store/prefix-selectors-with-select.ts

```diff
@@ -38,9 +46,52 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
             },
           },
           node: id,
-          messageId,
+          messageId: prefixSelectorsWithSelect,
+          suggest: [
+            {
+              messageId: prefixSelectorsWithSelectSuggest,
+              fix: (fixer) =>
+                fixer.replaceTextRange(
+                  [id.range[0], id.typeAnnotation?.range[0] ?? id.range[1]],
+                  suggestedName,
+                ),
+              data: {
+                name: suggestedName,
+              },
+            },
+          ],
         })
       },
     }
   },
 })
+
+function getSuggestedName(name: string) {
```

Comment:
So... this is a proposal... this function basically tries to deduce what the best name would be. Not sure if it's a good idea or if we should just append `select` + capitalize the first existent letter and that's fine or even better... if you have something in mind.

In any case, I'm all ears.

Reply:
I think that's a great idea! 
I say, keep it!